contracts/protocolmining/TimeLockProtocolMiningReward.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.8.0;
+pragma abicoder v2; //needed to return struct
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/access/TimelockController.sol";
+import "@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol";
+import "./FunctionCall.sol";
+import "../utils/Ownable.sol";
+
+/// @title TimeLockProtocolMiningReward smart contract
+/// @dev This contract is based on @openzeppelin/contracts/access/TimelockController.sol
+/// to use protocol mining with token reward.
+contract TimeLockProtocolMiningReward is Ownable
```

Comment:
`Ownable` is imported, but never used - neither `TimeLocker` nor `FunctionSelector` are `Ownable` either - must have been a premature import ?

Reply:
maybe, the last push might have had some "not in order" add/import.